remme/rpc_api/transaction.py

```diff
@@ -115,6 +155,24 @@
             message='Signer public key format is not valid'
         )
 
+    current_handler = None
+
+    for handler in TP_HANDLERS:
```

Comment:
Why not just build a dict here?

Reply:
Not in this place, but could be

Reply:
This might be better than iterating over a list :)

Reply:
I mean to place dict in more global place)